### PR TITLE
feat(i18n): updates translation for form/inputs/files after rebase of feat/i18n

### PR DESCRIPTION
**Pull request by jtpetty**
### Description

Updates translation for files in `packages/sanity/src/core/form/inputs/files`

### CODE REVIEW

File: packages/sanity/src/core/form/inputs/files/FileInput/InvalidFileWarning.tsx [hunk 1]
```diff
@@ -22,15 +24,17 @@ export function InvalidFileWarning({onClearValue}: Props) {
...
-            new file.
+            {t('inputs.file.invalid-file-warning.title')}
           </Text>
+          <Text size={1}>{t('inputs.file.invalid-file-warning.description')}</Text>
         </Stack>
       </Flex>
-      <ButtonWrapper icon={ResetIcon} text="Reset value" onClick={onClearValue} mode="ghost" />
+      <ButtonWrapper
+        icon={ResetIcon}
+        text={t('inputs.file.invalid-file-warning.reest-button.text')}
```
Comment (ricokahler) [inline packages/sanity/src/core/form/inputs/files/FileInput/InvalidFileWarning.tsx:34]: seems like there's a typo here `reest`

File: packages/sanity/src/core/i18n/bundles/studio.ts [hunk 2]
```diff
@@ -275,12 +302,46 @@ export const studioLocaleStrings = {
...
+
+  /** Invalid file value */
+  'inputs.file.invalid-file-warning.title': 'Invalid file value',
+
+  /** The value of this field is not a valid file. Resetting this field will let you choose a new file. */
+  'inputs.file.invalid-file-warning.description':
+    'The value of this field is not a valid file. Resetting this field will let you choose a new file.',
+
+  /** Reset value */
+  'inputs.file.invalid-file-warning.reest-button.text': 'Reset value',
```
Comment (ricokahler) [inline packages/sanity/src/core/i18n/bundles/studio.ts:337]: typo in `reest`?

**Review by ricokahler — APPROVED** [review]
i think i caught a typo but it looks good otherwise!

**Review by rexxars — APPROVED** [review]
Looks good aside from the typo Rico already pointed out 👍

**Pull request merged**